### ISSUE TITLE
Lack of home.txt

### ISSUE THREAD

**Issue by itwars** [mattrighetti/athome-scraper#2]
I great job! Could you just provide a example of home.txt?

Thx

**Comment by mattrighetti** [comment]
`home.txt` is just a bunch of links per-line 😄 something like this:

```
https://www.athome.lu/en/rent/apartment/luxembourg/id-7486621.html
https://www.athome.lu/en/rent/house/luxembourg/id-7511396.html
https://www.athome.lu/en/rent/house/strassen/id-7512843.html
```

**Comment by mattrighetti** [comment]
Fix in 0bc95a145e4da129d105f0938409b369bcf96289
